### PR TITLE
Update documentation to create template

### PR DESCRIPTION
**Pull request by carlh98**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Description

Update documentation to install and uninstall template

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [x] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
 
 We use `dotnet` project templates to easily create new projects. It simplifies the **project renaming** and supports **conditional inclusions**.
 
-### Installing and uninstalling the template
```
Comment (jeanplevesque) [inline README.md:9]: I wouldn't change this section.

File: README.md [hunk 2]
```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
...
+### Installing the template
 
-1. In order to install the template, clone this repository on your machine and open a command prompt at the root of the project and run the following command 
+1. In order to install the template, clone this repository on your machine and open a command prompt at the root of the cloned project and run the following command
```
Comment (jeanplevesque) [inline README.md:11]: "clone" is already mentioned.

It should be:
In order to install the template, clone this repository on your machine, open a command prompt at its root, and run the following command

File: README.md (hunk 1 above)
Comment (carlh98) [reply README.md:9]: It's confusing more than anything else, it look like steps to be done
1. Install
2. Uninstall

File: README.md [hunk 3]
```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
...
+1. In order to install the template, clone this repository on your machine, open a command prompt at its root, and run the following command
 
    `dotnet new -i ./`
 
-
-1. If you want to uninstall the template, run the following command.
-
-    `dotnet new -u ./`
-
     This will list you the list of installed templates, look for this template and copy the command with the absolute path like this. (Note the quotes added, otherwise it doesn't work)
```
[the remark this answers is not in the document]
Comment (carlh98) [reply README.md:15]: This text should stay in the Install section since it's to see the list of installed templates

File: README.md [hunk 4]
```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
...
 
     `dotnet new -u "C:\P\ApplicationTemplate"`
 
 ### Running the template to generate a new project
 
-1. To run the template and create a new project, run the following command.
+1. To run the template and create a new project, run the following command in the root folder that will contain the new project.
```
Comment (Guidemarcus) [inline README.md:21]: run the following command AT the root folder that will contain the new project

**Review by Guidemarcus — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (carlh98) [reply README.md:21]: Verified with Ryan and "in" implies that the user should be inside the folder while running the command

**Pull request merged**